### PR TITLE
add badges for license and CI/CD

### PR DESCRIPTION
**Pull request by woutdenolf**
Add badges for CI, CD and license:
image

### CODE REVIEW

File: README.md
Comment (PeterC-DLS) [inline README.md:8]: Not https and do we still want to use this rather than GH actions?

Comment (woutdenolf) [reply README.md:8]: So I remove `Build Server: http://build.nexusformat.org/`? And what about `[![Travis CI Status] ...`?

Comment (PeterC-DLS) [reply README.md:8]: Yes, that too, please!

Comment (prjemian) [reply README.md:8]: Note that https://nexusformat.github.io/definitions (the default GH Actions publishing URL) redirects to https://manual.nexusformat.org/ -- try it!

Comment (prjemian) [reply README.md:8]: no travis-ci Since #868

Comment (prjemian) [reply README.md:8]: Others are good, thanks for the contributions!

Comment (prjemian) [reply README.md:8]: Since we added GH Actions CI workflows, the old CI build server (http://build.nexusformat.org/) is legacy and I suggest we do not refer to it any longer.

File: README.md
Comment (prjemian) [inline README.md:3]: This is not needed.  Let's leave it *branded* to nexusformat.org.

Comment (woutdenolf) [reply README.md]: Ah ok, I misunderstood your comment. I put it back to the original URL.

**Comment by woutdenolf** [comment]
Reviewed version:
image

**Comment by prjemian** [comment]
LGTM!  I'll wait for review from @FreddieAkeroyd before adding mine.

**Review by PeterC-DLS — APPROVED** [review]
Nice!

**Comment by woutdenolf** [comment]
> LGTM! I'll wait for review from @FreddieAkeroyd before adding mine.

@FreddieAkeroyd Would you have some time to take a look at this PR?

**Review by FreddieAkeroyd — APPROVED** [review]
Looks good

**Pull request merged**
